gpbft/src/powertable.rs

```diff
@@ -37,7 +38,34 @@ impl Ord for PowerEntry {
     }
 }
 
+impl Serialize for PowerEntry {
```

Comment:
Possible to use `serde_tuple` to avoid the manual implementations?

Reply:
There is a validation there as well to match the Go code. But we can at least derive deserialize.